### PR TITLE
fix: Run single request for SDKs

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

File: src/gatsby/utils/packageRegistry.ts [hunk 1]
```diff
@@ -45,28 +43,16 @@ export default class PackageRegistry {
...
-    return this.cache[name];
+    await this.getList();
+    return this.indexCache[name]
   };
 
-  version = async (name, defaultValue: string = "") => {
+  version = async (name: string, defaultValue: string = "") => {
     const data = (await this.getData(name)) as VersionData;
-    return data.version || defaultValue;
+    return data && data.version || defaultValue;
```
Comment (BYK) [inline src/gatsby/utils/packageRegistry.ts:52]: Are you TS yet?

```suggestion
    return data?.version || defaultValue;
```

**Review by BYK — APPROVED** [review]
gogogo

File: src/gatsby/utils/packageRegistry.ts (hunk 1 above)
Comment (untitaker) [reply src/gatsby/utils/packageRegistry.ts:52]: i'm in fact not ts yet, doing this breaks

i think we're running an old ts

Comment (BYK) [reply src/gatsby/utils/packageRegistry.ts:52]: or maybe I don't know how to ts 😁

Comment (untitaker) [reply src/gatsby/utils/packageRegistry.ts:52]: no I tried before, and all the stuff we can do in sentry frontend doesn't work here

**Pull request merged**